src/go/rpk/pkg/cli/cmd/root.go

```diff
@@ -425,3 +430,45 @@ func walk(c *cobra.Command, f func(*cobra.Command)) {
 		walk(c, f)
 	}
 }
+
+func wrappedLocalFlagUsages(cmd *cobra.Command) string {
+	width := 80
+	if ws, err := mTerm.GetWinsize(0); err == nil {
+		width = int(ws.Width)
+	}
+	return cmd.LocalFlags().FlagUsagesWrapped(width - 1)
```

Comment:
Why width-1? What happens at 0 width?

Reply:
Cool. After flipping between the two pictures, I'm not sure the -1 is necessary. wdyt about removing?